api/src/legacy_soap_api/legacy_soap_api_client.py

```diff
@@ -56,7 +57,9 @@ def get_soap_response_dict(self) -> dict:
         Example:
             {'Envelope': {'Body': {'GetOpportunityListResponse': {...}}}}
         """
-        operation_method = getattr(self, self.operation_config.request_operation_name)
+        operation_method = getattr(
+            self, to_snake_case(self.operation_config.request_operation_name)
+        )
```

Comment:
Ah, hadn't realized that the function name was being used in that way, but good way of fixing it. Alternatively could have a mapping config to the function, but this works.